xml/repeated-content-decl.ent

```diff
@@ -1989,8 +1989,9 @@ User:           username@ldap.example.com
 
 <!ENTITY values-file-changes
 '<para xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
-  The format of the &values-filename; file has been restructured completely.
-  Do not re-use the previous version of the file. Instead, source the default file from the appendix in
+  The format of the &values-filename; file has been restructured completely in
+  &cap; 2.0. Do not re-use the &cap; 1.x version of the file. Instead, source the
```

Comment:
For the first sentence, any opinions on changing it to `2.x` since it's applicable to more than just 2.0 now?

For the second sentence, could you rephrase the part about sourcing the file. We want to avoid possible misunderstandings where people are just copy and pasting the whole thing and being overwhelmed.

Reply:
I have updated the wording. Let me know if it still needs improvement.